### ISSUE TITLE
When using the command palette to execute Profiler: Create Profiler Session a generic error occurs

### ISSUE THREAD

**Issue by ConstantineK** [Microsoft/azuredatastudio#3325]
- Azure Data Studio Version:
Version 1.3.3-insider
Commit c806cc5ff1931237aee0c518e4ca3661d38036ff
Date 2018-11-27T07:50:22.627Z
VS Code 1.26.1
Shell 2.0.9
Renderer 61.0.3163.100
Node 8.9.3
V8: 6.1.534.41
Architecture x64

Steps to Reproduce:

1. Install insider build from https://github.com/Microsoft/azuredatastudio/releases
2. Install the most recent SQL Server Profiler extension from Microsoft 
3. Reload ADS to ensure the extension is available
4. Open the command palette (I am on windows so I used ctrl+shift+p) and type Profiler
5. Choose the first option Profiler: Create Profiler Session 
6. An error occurs - "Running the contributed command:'profiler.openCreateSessionDialog' failed."

![2018-11-27_15-41-21](https://user-images.githubusercontent.com/6138430/49119648-f91b5b00-f25d-11e8-98f4-9a1e65eca690.gif)

**Comment by ConstantineK** [comment]
Sorry I should have added, it 100% works once the connection is setup, so this issue only is important if you want a better error message or you want to pop up the dialog.

**Comment by yualan** [comment]
Yes, there is a plan to pop up connection dialog. If you press Alt+P when you don't have any connections, it pops connection dialog. Looks like we need to add that also for command palette. Thanks for reporting!

**Comment by alanrenmsft** [comment]
Ack, I will take a look at this.

**Comment by yualan** [comment]
Thanks, as @alanrenmsft investigated, the Launch Profiler command is the intended launch point from the Command Palette (in addition to Alt+P and right-click). This already will pull up the Connection Dialog if no active connections.

For the other Profiler commands, the existing commands for start/stop/create session are a bit confusing in the command palette, so to help avoid confusion, we will remove these commands from the palette.
